js/actions/appActions.js

```diff
@@ -1411,6 +1411,14 @@ const appActions = {
     dispatch({
       actionType: appConstants.APP_UPDATE_LOG_OPENED
     })
+  },
+
+  updateComputedStyles: function (windowId, values) {
```

Comment:
I think this should actually be in windowActions. We are generally moving away from windowActions and appActions, but we will keep windowActions for actions that are related to the window component. It can still be handled in the browser process